addons/account/models/chart_template.py

```diff
@@ -73,8 +73,7 @@ def _avoid_name_conflict(company, template):
             _avoid_name_conflict(company, template)
             templates_to_create += template
         new_template2tax_company = templates_to_create._generate_tax(company, accounts_exist=True)['tax_template_to_tax']
-        return [(env['account.tax.template'].browse(template_id), env['account.tax'].browse(tax_id))
-                for template_id, tax_id in new_template2tax_company.items()]
+        return [(template_id, tax_id) for template_id, tax_id in new_template2tax_company.items()]
```

Comment:
Pretty sure this is equivalent to`return new_template2tax_company.items()` ?

Reply:
yes you're right